testconvolute.m

```diff
@@ -1,6 +1,43 @@
-xt = ones(1,100);
-ht = 0.01:0.01:2;
-a = length(xt);
-Z = convolute(xt,ht)
-Z(1,1) = Z(1,1)
-a = conv(0.01*xt,ht)
\ No newline at end of file
+xt = ones(1,100);%function x(t)
+ht = 0.01:0.01:2;%function h(t)
+Zt = convolute(xt,ht);%function Z(t), convoluted function
+cCheck = conv(0.01*xt,ht);%Our check for our test
+Zt %comment out if you want to print @MickeyDang

```

Comment:
These comments will be submitted so remove them. Just state that the output is printed and can be manually compared to expected value.